src/main/java/org/terasology/scenario/components/information/BlockComponent.java

```diff
@@ -20,6 +20,9 @@
 import org.terasology.network.Replicate;
 import org.terasology.world.block.family.BlockFamily;
 
+/**
+ * Component that includes a block family

```

Comment:
It would be nice if this javadoc would give a hint what this component is used for. e.g. which entity/prefab contains it normally. That it contains a block family is just visible below.